### ISSUE TITLE
Links in readme generate 404s

### ISSUE THREAD

**Issue by CoolDadTx** [APIPatterns/APIPracticePatterns#3]
Need to check your links in the readme. The top level links for your sections generate 404s. The documents seem to be there so it looks like a pathing issue. The subsections seem to work OK.

**Comment by markweitzel** [comment]
Yep! Thanks! I'm cleaning these up now and standing up a gh pages site.

**Comment by markweitzel** [comment]
@CoolDadTx -- take a peek at https://github.com/APIPatterns/APIPracticePatterns 
everything should be fixed now.

**Comment by CoolDadTx** [comment]
Yes they work now. Thanks.
